scripts/cibuild

```diff
@@ -23,16 +23,33 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     if [ "${1:-}" = "--help" ]; then
         usage
     else
-        ./scripts/format
-        ./scripts/lint
-        ./scripts/test
+        #Update dependencies
+        echo "Updating frontend dependencies"
+        docker-compose \
+            run --rm client \
+            yarn install
+
+        #Build static asset bundle
+        echo "Building static bundle"
+        docker-compose \
+            run --rm client \
+            yarn run build
 
-        # Build static asset bundle
+        # Bring up PostgreSQL in a way that respects configured
+        # service health checks.
         docker-compose \
-            run --rm --no-deps \
-            -e NODE_ENV="${NODE_ENV:-production}" \
-            -e INSTALL_ENV="${INSTALL_ENV:-internal}" \
-            -e REACT_APP_GIT_COMMIT="${GIT_COMMIT}" client \
-            yarn build
+            -f docker-compose.yml \
+            up -d database
+
+        # Build tagged container images
+        GIT_COMMIT="${GIT_COMMIT}" docker-compose \
```

Comment:
Do not need to set GIT_COMMIT var since it's not being used in the following script files